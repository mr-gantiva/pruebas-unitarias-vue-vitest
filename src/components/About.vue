<script setup>

</script>

<template>
    <div>
        <h1>This is an about page</h1>
        <div>
            <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec
                fermentum nunc. Nullam nec fermentum nunc. Nullam nec fermentum nunc.
                Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec
            </p>
        </div>
    </div>
</template>



<style scoped></style>