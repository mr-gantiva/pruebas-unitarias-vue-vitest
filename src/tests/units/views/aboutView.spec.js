import { shallowMount } from '@vue/test-utils'
import About from '@/components/About.vue'
import { describe, expect, test } from 'vitest'

describe('Componente About.vue', () => {
  test('Snapshot de About.vue', () => {
    // Selección del sujeto de pruebas
    const wrapper = shallowMount(About)
    // Validación de match con el snapshot
    expect(wrapper.html()).toMatchSnapshot()
  })
})
