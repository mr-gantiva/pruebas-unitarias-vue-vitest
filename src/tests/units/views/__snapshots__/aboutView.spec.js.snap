// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Componente About.vue > Snapshot de About.vue 1`] = `
"<div>
  <h1>This is an about page</h1>
  <div>
    <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec </p>
  </div>
</div>"
`;

exports[`Componente About.vue > Validaci√≥n de match con el snapshot 1`] = `
"<div>
  <h1>This is an about page</h1>
  <div>
    <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec fermentum nunc. Nullam nec </p>
  </div>
</div>"
`;
